docs/services/table-of-contents-scroll-spy.service.spec.ts

```diff
@@ -0,0 +1,78 @@
+/*!
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import {TestBed, discardPeriodicTasks, fakeAsync, tick} from '@angular/core/testing';
```

Comment:
are we not running these specs? I worry we'll forget doing this in the future, if we don't do it now?

Reply:
I knew there was something on my todo list for this branch...

Reply:
Thanks! can you also provide additional context on the reason for this in the commit message? would be good to capture this just for the future (and maybe for me :D )

Reply:
Done.